docs/content.md

```diff
@@ -4,83 +4,79 @@
 
 ###  Global
 
-* Design Principles
-* Accessibility
-* Voice & tone (off link to Manual of Style?)
-* Designing for Safety
+* Accessibility: consider a11y in design early, list of tools
+* Design principles
+* Designing for safety
+* Content
 
-### Foundation
+### Foundations
 
-* Color: palette, use/brand (use more, use less), a11y (recommended combos)
-* Typography: scale, use (bold, normal, light, sans, mono) semantic markup/a11y,
+* Color: palette, use/brand (use more, use less), a11y (recommended combos), accent color
 * Spacing: scale, density on product vs spacious on marketing pages
+* Typography: scale, use (bold, normal, light, sans, mono) semantic markup/a11y
 
-### UI Patterns
+### Layout
+
+* Navigation
+* Responsive, desktop & mobile first
+* Templates: page layouts & hierarchy
 
+### UI patterns
+
+* Animation
+* Button usage
+* Defensive design: valley of safety vs destructive, destructive actions, danger
 * Empty states (blankslates)
-* Saving models
 * Loading states
-* Animation
-* Progressive disclosure (show more, expand etc.)
-* Defensive design (valley of safety vs destructive)
+* Messaging
+* Progressive disclosure: show more, expand, etc.
+* Search and filter
```

Comment:
Would this section include guidelines around filtering vs sorting?

Reply:
I'm not 100% sure how to separate this section from tables and lists in a way that people can find what they're looking for easily. Searching and filtering always seems to happen within tables and lists, right? Another thought I just had: in the Workflows section I include things that can be defined with action verbs (Creating, Onboarding, Saving), so maybe Searching (and filtering) are part of Workflows? 🤔